tests/ordering/tests.py

```diff
@@ -212,14 +218,9 @@ def test_order_by_pk(self):
         """
         'pk' works as an ordering option in Meta.
         """
-        Author.objects.create(pk=1)
-        Author.objects.create(pk=2)
-        Author.objects.create(pk=3)
-        Author.objects.create(pk=4)
-
         self.assertQuerysetEqual(
             Author.objects.all(), [
-                4, 3, 2, 1
+                5, 4, 3, 2, 1
```

Comment:
Does the next patch use the fifth `Author`? If so, it might be more maintainable to replace this hardcoded list with something like `list(range(1, Author.objects.count() + 1))`